base/loading.jl

```diff
@@ -947,7 +965,11 @@ function _require(pkg::PkgId)
         name = pkg.name
         path = locate_package(pkg)
         if path === nothing
-            throw(ArgumentError("Module $name not found in current path.\nRun `Pkg.add(\"$name\")` to install the $name package."))
+            throw(ArgumentError("""
```

Comment:
Again, perhaps just recommend `Pkg.instantiate()` instead of the repl.

Reply:
For the future we should also check if we failed to find an explicit path in the manifest  and print that one but could be added later.